### PR TITLE
Hard code users-meta db replication

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

We rely on this replication being enabled so it shouldn't be
configurable. This change means it just works with no configuration
required.

medic/cht-core#6519

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: sentinel/src/schedule/index.js [hunk 1]
```diff
@@ -13,67 +12,54 @@ const tasks = {
...
-      _reschedule();
-    }
-  );
+  tasks.reminders.execute()
+    .then(() => executeIfSendable(tasks.dueTasks))
+    .then(() => tasks.replications.execute())
+    .then(() => tasks.outbound.execute())
+    .then(() => tasks.purging.execute())
+    .catch(err => logger.error('Error running tasks: %o', err))
+    .then(() => reschedule());
```
Comment (dianabarsan) [inline sentinel/src/schedule/index.js:64]: Such an eye sore cleared. Big win!!

File: sentinel/tests/unit/schedule/reminders.js [hunk 2]
```diff
@@ -20,24 +21,21 @@ describe('reminders', () => {
     db.couchUrl = 'someURL';
   });
   beforeEach(() => {
-    process.env.TEST_ENV = true;
+    process.env.TEST_ENV = true; // TODO remove?
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply sentinel/tests/unit/schedule/reminders.js]: Removed along with a few other examples...

File: sentinel/src/schedule/replications.js [hunk 3]
```diff
@@ -141,40 +113,20 @@ function replicateDb(sourceDb, targetDb) {
...
-        const timer = later.setInterval(() => module.exports.runReplication(replication), sched);
-        timers.push(timer);
-        return Promise.resolve();
-      });
-    }, Promise.resolve())
-      .then(() => callback())
-      .catch(callback);
+  execute: () => {
+    if (timer) {
+      timer.clear();
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply sentinel/src/schedule/replications.js]: Good call. Removed.

File: sentinel/src/schedule/index.js (hunk 1 above)
Comment (garethbowen) [reply sentinel/src/schedule/index.js:64]: I couldn't resist...

**Pull request merged**